modules/apps/commerce/commerce-api/src/main/java/com/liferay/commerce/order/engine/CommerceOrderEngine.java

```diff
@@ -47,6 +51,11 @@ public CommerceOrder transitionCommerceOrder(
 			CommerceOrder commerceOrder, int orderStatus, long userId)
 		throws PortalException;
 
+	public CommerceOrder transitionCommerceOrder(
```

Comment:
don't create new methods, replace the old one and just put the secure value true as default

Reply:
Updated PR before I go to sleep @luca-pellizzon .